src/sentry/incidents/action_handlers.py

```diff
@@ -90,7 +90,9 @@ def email_users(self, status):
     def build_message(self, context, status, user_id):
         display = self.status_display[status]
         return MessageBuilder(
-            subject=u"Alert Rule {} for Project {}".format(display, self.project.slug),
+            subject=u"[{}] {} - {}".format(
+                context["status"], context["incident_name"], self.project.slug
+            ),
             template=u"sentry/emails/incidents/trigger.txt",
             html_template=u"sentry/emails/incidents/trigger.html",
             type="incident.alert_rule_{}".format(display.lower()),
```

Comment:
What is this used for? I couldn't really trace it, doesn't seem like it's needed?

Reply:
Many `MessageBuilder` uses include type. I don't know specifically what it's used for, going to assume it's somehow stat related.